### PR TITLE
RESOURCE-118 Support azure service fabric mesh apps

### LINKED ISSUE #570: Service Fabric mesh Apps
https://docs.microsoft.com/en-us/rest/api/servicefabric/sfmeshrp-index-applications

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description
Support azure service fabric mesh app(s)

### Issues Resolved
#570 

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by soumyo13 — APPROVED** [review]
LGTM

**Review by soumyo13 — CHANGES_REQUESTED** [review]
Terraform files are missing.

File: test/integration/verify/controls/azure_service_fabric_mesh_application.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+resource_group = input(:resource_group, value: '')
+location = input(:location, value: '')
+
+control 'test the properties of an Azure Service Fabric Mesh Application' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_application.rb:4]:
```suggestion
skip_control 'test the properties of an Azure Service Fabric Mesh Application' do
```

File: test/integration/verify/controls/azure_service_fabric_mesh_applications.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+location = input(:location, value: '')
+
+control 'test the properties of all Azure Service Fabric Mesh Applications' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_applications.rb:3]:
```suggestion
skip_control 'test the properties of all Azure Service Fabric Mesh Applications' do
```

**Review by soumyo13 — APPROVED** [review]

**Review by IanMadd — APPROVED** [review]

**Pull request merged**